server/services/publication-service.js

```diff
@@ -70,6 +72,7 @@ export default ({ strapi }) => ({
 		if (mode === 'publish' && ((!isPublished && isDraft) || isModified)) {
 			await this.publish(record.entitySlug, entityId, {
 				publishedAt: record.executeAt ? new Date(record.executeAt) : new Date(),
+				locale: record.locale,
```

Comment:
If we add the locale to the publish function, should we add it to the unpublish function too?